lib/tasks/cards.rake

```diff
@@ -222,45 +228,305 @@ namespace :cards do
     }
   end
 
+  def import_formats(formats_json)
+    formats = []
+    formats_json.each { |f|
+      active_id = nil
+      f['snapshots'].each_with_index do |s, i|
+        next if s['active'].nil?
+        raise 'Multiple snapshots marked active in format %s.' % f['name'] unless !active_id
```

Comment:
The table can only have one active snapshot anyway. I think when I added this check it was to warn the user that they were probably doing something wrong rather than protect the database from bad data. We could add something to the json schema or just change the way it represents this.